README.md

```diff
@@ -1,2 +1,69 @@
 # fastify-get-head
-Fastify plugin to set a HEAD route handler for each GET handler previously registered
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/fastify/fastify-cookie.svg)](https://greenkeeper.io/)
+
+<!-- [![Build Status](https://travis-ci.org/fastify/fastify-cookie.svg?branch=master)](https://travis-ci.org/fastify/fastify-cookie) -->
+
+A plugin for [Fastify](http://fastify.io/) that adds support for setting a `HEAD` route for each `GET` one previously registered.
+
+This plugin's cookie parsing works via Fastify's `onRoute` hook. When a new route is registered, the plugin will try to set a new `HEAD` route if the registered route is for a `GET` method.
+
+`fastify-cookie` v3 only supports Fastify@3.x.
```

Comment:
This is out of date

Reply:
Sure, just to double-check, I think the plugin is coupled to the version where this [commit](https://github.com/fastify/fastify/commit/449fc5c25e7d2e277e68bb3f4416cfd858a71cec) is released, isn't it?